Makefile

```
@@ -108,17 +111,9 @@ dev.pull.%: ## Pull latest Docker images for a given service and all its depende
 
 dev.up: | check-memory ## Bring up all services with host volumes
 	bash -c 'docker-compose $(STANDARD_COMPOSE_FILES) up -d'
-	@# Comment out this next line if you want to save some time and don't care about catalog programs
-	$(WINPTY) bash ./programs/provision.sh cache
 
 dev.up.%: | check-memory ## Bring up a specific service and its dependencies with host volumes
 	bash -c 'docker-compose $(STANDARD_COMPOSE_FILES) up -d $*'
-	@# Comment out this next line if you want to save some time and don't care about catalog programs
-ifdef no_cache
-	echo "Not runing bash ./programs/provision.sh cache"
-else
-	$(WINPTY) bash ./programs/provision.sh cache
-endif
 
```

```suggestion
dev.up.withprograms.%: dev.up.$* dev.cache-programs  ## Bring up a service + its dependencies and cache programs in LMS.

dev.up.withprograms: dev.up dev.cache-programs  ## Bring up all services and cache programs in LMS.
```

Regarding the suggestion in your response to @cpennington , which I liked.

---

As someone who frequently needs to have those programs cached in the LMS, I'd prefer to have both this explicit `dev.up.with-programs` make target AND an environment variable that makes `dev.up` run the program-caching command as it currently does.  To me, the risk introduced by the implicitness of an environment variable's state is small enough to accept.

---

I obviously sympathize with your sentiment @iloveagent57 , but as we're in the minority that actually benefits from caching programs, maybe we should roll that change ourselves in a separate PR.

---

👍 sounds good to me